packages/shared-base/package.json

```diff
@@ -21,10 +21,12 @@
     "immer": "^9.0.12",
     "pvtsutils": "^1.2.2",
     "react": "../empty",
+    "react-use": "^17.3.2",
```

Comment:
hello, please don't do this. shared-base should never export a react class. it's pure and can be run in any environment.

Reply:
or use any react feature. as you can see, I explicitly resolves `react` to an empty package within this `shared-base` package.

Reply:
I have to add another package for shared-base UI.